packages/salesforcedx-utils-vscode/src/commands/preconditionCheckers.ts

```diff
@@ -27,7 +27,7 @@ export class IsSalesforceProjectOpened implements Predicate<typeof workspace> {
     ) {
       return PredicateResponse.of(
         false,
-        nls.localize('predicates_no_sfdx_project_found_text')
+        nls.localize('predicates_no_sf_project_found_text')
```

Comment:
I think the acronym "sf" is fine when it's lowercase and/or used as variable names, such as this case.  (I guess lowercase "sf" makes me think of the CLI command that I type in the Terminal instead of San Francisco?)